presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/api/ConnectorsAPI.java

```diff
@@ -33,38 +34,64 @@
 import java.nio.file.Paths;
 
 @Path("/connectors")
```

Comment:
Since we have other outstanding PRs , I would change this after all is merged. I will put this into TODO.